src/c++/perf_analyzer/genai-perf/tests/test_cli.py

```diff
@@ -758,20 +786,6 @@ def test_compare_not_provided(self, monkeypatch, capsys):
         captured = capsys.readouterr()
         assert expected_output in captured.err
 
-    @pytest.mark.parametrize(
```

Comment:
Why remove this?

Reply:
Great question. This test tests that the model argument is None for compare.

In its current state, compare gets all of the args that we use for profile parsing. The model name for compare should be None, not the default model name. With this change, compare will only have the args that its parser adds and the compare subcommand needs. So we no longer need to test for this workaround.